docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Multi-Assets | Dagster
+description: A multi-asset represents a set of assets that are all updated by the same operation.
+---
+
+# Multi-Assets
+
+A multi-asset represents a set of assets that are all updated by the same operation.
+
+## Relevant APIs
+
+| Name                                        | Description                              |
+| ------------------------------------------- | ---------------------------------------- |
+| <PyObject object="multi_asset" decorator /> | A decorator used to define multi-assets. |
+
+## Overview
+
+When working with software-defined assets, it's sometimes inconvenient or impossible to map each persisted asset to a unique operation. A multi-asset is a way to define a single operation that will produce the contents of multiple data assets at once. Multi-assets may be useful in the following scenarios:
+
+- A single call to an API results in multiple tables being updated (e.g. Airbyte, Fivetran, dbt)
+- A DataFrame is transformed into multiple different assets, and serializing it between separate ops would be prohibitively inefficient
+- The same in-memory object is used to compute multiple different assets
+
+## Defining multi-assets
+
+The function responsible for computing the contents of any software-defined asset is an [op](/concepts/ops-jobs-graphs/ops). Multi-assets are responsible for updating multiple assets, so the underlying op will have multiple outputs -- one for each associated asset.
+
+### A basic multi-asset
+
+The easiest way to create a multi-asset is with the <PyObject object="multi_asset" decorator /> decorator. This decorator functions similarly to the <PyObject object="asset" decorator /> decorator, but requires an `outs` parameter specifying each output asset of this function.
+
+```python file=/concepts/assets/multi_assets.py startafter=start_basic_multi_asset endbefore=end_basic_multi_asset
+from dagster import multi_asset, Out
+
+
+@multi_asset(
+    outs={
+        "my_string_asset": Out(),
+        "my_int_asset": Out(),
+    }
+)
+def my_function():
+    return "abc", 123
+```
+
+By default, the names of the outputs will be used to form the asset keys of this multi-asset. The decorated function will be used to create the op for these assets, and must emit an output for each of them. In this case, we can emit multiple outputs by returning a tuple of values, one for each asset.
```

Comment:
> The decorated function will be used to create the op for these assets, and must emit an output for each of them.

Let's remove the comma here since it isn't necessary.